cmd/katapult/data_centers.go

```diff
@@ -45,17 +47,14 @@ func dataCentersCmd(client *katapult.Client) *cobra.Command {
 		Short: "Get details for a data center",
 		Long:  "Get details for a data center.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			dc, resp, err := client.DataCenters.Get(cmd.Context(), args[0])
+			dcClient := core.NewDataCentersClient(client)
+			dc, resp, err := dcClient.Get(cmd.Context(), core.DataCenterRef{Permalink: args[0]})
 			if err != nil {
-				if resp == nil || resp.StatusCode != 404 {
-					return err
-				}
-
-				dc, _, err = client.DataCenters.GetByPermalink(
-					cmd.Context(), args[0])
-				if err != nil && resp.StatusCode != 404 {
-					return err
+				if resp != nil && resp.StatusCode == 404 {
+					fmt.Printf("Unknown datacentre.")
+					return nil
```

Comment:
Hmm, perhaps we want to return this as an error with that message instead ? Otherwise the exit code will be wrong.